### ISSUE TITLE
Agree on Statistics Context

### ISSUE THREAD

**Issue by patricktoohey** [patricktoohey/packml-hmi#34]
Statistics is the preferred term in all prototypes. Icon is a line chart, not a bar graph. Additional adornments should not be present (e.g., borders, axes). Scope is device specific.

**Comment by patricktoohey** [comment]
Position Statistics icon to the right of the Products icon.

**Comment by BerryJansen** [comment]
Agree on the icon. 
Are the OEE and state/mode times in the statistics scope?

**Comment by patricktoohey** [comment]
I agree that statistics is a good place for this data. Perhaps dual-homed with the Machine Control section as well. State pareto diagram is an effective startup utility.

**Comment by patricktoohey** [comment]
SPC as well
